incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -50,12 +50,14 @@ type ObjectReconciler struct {
 // +kubebuilder:rbac:groups=*,resources=*,verbs=get;list;watch;create;update;patch;delete;deletecollection;use;impersonate
 
 func (r *ObjectReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	if !ex[req.Namespace] {
-		atomic.AddInt32(&obTot, 1)
-		atomic.AddInt32(&obCur, 1)
-		defer atomic.AddInt32(&obCur, -1)
+	if ex[req.Namespace] {
+		return ctrl.Result{}, nil
 	}
 
+	stats.Stats.ObTot.Incr()
+	stats.Stats.ObCur.Incr()
+	defer stats.Stats.ObCur.Decr()
```

Comment:
It's definitely more observable to have numbers by object types, but do we just need the logs for performance testing? Since the test case will only have 1 type created in all namespaces.

Reply:
Just discussed this offline but for the record - we'll probably use these same calls to generate metrics so it makes sense to group them by GVK too. Actually, on second thought, we don't really need the version, so just group+kind is probably best.

Reply:
After discussion offline, I will break it by object type for not only the performance testing but also later metrics.